composer.json

```diff
@@ -5,15 +5,15 @@
   "homepage": "https://github.com/calcinai/xero-php",
   "license": "MIT",
   "require": {
-    "php": ">=5.5.0",
+    "php": ">=8.0.0",
```

Comment:
Thought we should probably move to a php version that's even close to being supported - https://www.php.net/supported-versions.php
From memory someone got the package ready for 8.1. 
To try and run tests originally i had to scour the internet for a PHP 5 windows installer.

Reply:
I was _going_ to get it ready for 8.1 but it hasn't happened yet so I'd skip this one for now

Reply:
@Healyhatman You want me to revert the composer changes or leave them as is?

Reply:
It's @calcinai 's repo but I expect he wouldn't want a major breaking change like that ? I mean anyone still on 5.5 deserves what they get but still....

Reply:
Hey team, I've been investigating how to do this gracefully, and I think composer is smart enough to resolve dependencies back to a previous supported version.  I think @Healyhatman's changes can go through on main, and we can make a branch to support PHP5.

Reply:
@calcinai  Agree, composer will install the earliest possible version that's compatible. Only reason i did this is because i couldn't get tests to run without doing a crazy amount of messing around to get a php 5 to install locally. 

https://stitcher.io/blog/php-version-stats-january-2023 - PHP 8+ is over 50% of usage now which is great for the ecosystem. According to this site 7.0 and below each have less than 1% representation.